lib/season_stat_coach.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'game_collection'
+require_relative 'game_team_collection'
+require_relative 'team_collection'
+
+class SeasonStatCoach
+
```

Comment:
this is the coaches class, it has all the csvs right now, i'll remove everything but the game_team_collection in a later refactor because I only need the game_team collection